.travis.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+branches:
+  only:
+    - master
+
+language: php
+
+matrix:
+  include:
+    - php: 5.5
+      env: RUN_DEPLOYMENT_TESTS=true RUN_DEVSERVER_TESTS=true RUN_CS_FIXER=true
```

Comment:
Makes sense, I think we should just remove the commented-out versions then. And also remove the `matrix`, since we don't need it:

```yaml
php:
  - 5.5
env:
  global:
    - RUN_DEPLOYMENT_TESTS=true
    - RUN_DEVSERVER_TESTS=true
    - RUN_CS_FIXER=true
```